apps/web/src/views/Swap/V3Swap/containers/SwapCommitButton.tsx

```diff
@@ -334,11 +335,17 @@ export const SwapCommitButton = memo(function SwapCommitButton({
           id="swap-button"
           disabled={!isValid || !approved || (priceImpactSeverity > 3 && !isExpertMode)}
         >
-          {priceImpactSeverity > 3 && !isExpertMode
-            ? t('Price Impact High')
-            : priceImpactSeverity > 2
-            ? t('Swap Anyway')
-            : t('Swap')}
+          {(tradeLoading && (
+            <>
+              {t('Searching For The Best Price...')}
+              <Loading width="12px" height="12px" ml="4px" />
```

Comment:
Let's remove the `Loading` and add Dots component instead of hardcoded
`<Dots>Searching For The Best Price</Dots>`